crates/cheatcodes/src/inspector.rs

```diff
@@ -264,12 +283,6 @@ pub struct Cheatcodes {
     /// execution block environment.
     pub block: Option<BlockEnv>,
 
-    /// The gas price
-    ///
-    /// Used in the cheatcode handler to overwrite the gas price separately from the gas price
-    /// in the execution environment.
-    pub gas_price: Option<U256>,
```

Comment:
idk if this makes sense to be moved, this is just used in initialize_interpreter to set env

Reply:
makes sense, moved back in https://github.com/foundry-rs/foundry/pull/8750/commits/27f02ecf6a6e429f3fc3a7b2daf77eaf3da096a7